warehouse/transformer/benchmark/Makefile

```diff
@@ -0,0 +1,52 @@
+REQUIRED_VARS_BUILD = DOCKER_USER
+REQUIRED_VARS_DEPLOY = DOCKER_USER K8S_NAMESPACE MODE
+
+define check_vars
+    $(foreach var,$(1),$(if $(value $(var)),,$(error $(var) is not set)))
+endef
+
+ifeq ($(MAKECMDGOALS),build-transformer)
+    $(call check_vars,$(REQUIRED_VARS_BUILD))
+endif
+
+ifeq ($(MAKECMDGOALS),build-push-transformer)
+    $(call check_vars,$(REQUIRED_VARS_BUILD))
+endif
+
+ifeq ($(MAKECMDGOALS),prepare-deployment)
+    $(call check_vars,$(REQUIRED_VARS_DEPLOY))
+endif
+
+ifeq ($(MAKECMDGOALS),deploy-transformer)
+    $(call check_vars,$(REQUIRED_VARS_DEPLOY))
+endif
+
+ifeq ($(MAKECMDGOALS),delete-transformer)
+    $(call check_vars,$(REQUIRED_VARS_DEPLOY))
+endif
+
+IMAGE_NAME = $(DOCKER_USER)/rudder-warehouse-benchmarks:latest
+DOCKERFILE_PATH = ./warehouse/transformer/benchmark/Dockerfile
```

Comment:
Can `DOCKERFILE_PATH` be a relative path - `./Dockerfile` ?

Reply:
No. We need to build the image from the root directory. That's why we are doing this.
```
build-transformer:
	cd ../../../ && \
	docker buildx build -t $(IMAGE_NAME) -f $(DOCKERFILE_PATH) --platform=$(PLATFORM) .
```